03_letters/srini/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	letterMap := make(map[rune]int)
+
+	for _, r := range s {
+		if _, ok := letterMap[r]; ok {
+			letterMap[r]++
```

Comment:
I think you should be ok to just use `letterMap[r] += 1` as it is initialised to `0`

Reply:
I added that but I get this suggestion "[go-lint] should replace letterMap[r] += 1 with letterMap[r]++"

Reply:
also good - the point is: I don't think you need the `if/else` as `letterMap[r]` is initialised to `0`